website/tantalus/signals.py

```diff
@@ -0,0 +1,14 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from orders.models import Shift
+from tantalus.models import TantalusShiftSynchronization
+from tantalus.services import synchronize_to_tantalus
+
+
+@receiver(post_save, sender=Shift)
+def sync_to_tantalus(sender, instance, **kwargs):
+    """Synchronize Orders to Tantalus if a Shift is made finalized."""
+    if instance.finalized and not TantalusShiftSynchronization.objects.get(shift=instance).exists():
+        if synchronize_to_tantalus(instance):
+            TantalusShiftSynchronization.objects.create(shift=instance)
```

Comment:
I think creating the TantalusShiftSynchronization should also move to synchronize_to_tantalus

Reply:
I thought about this but it might also be needed to do a synchronize without creating an instance of that class. We can for example add the creation of an instance when creating a management command.

Reply:
Sure